.gitlab-ci.yml

```diff
@@ -536,6 +536,12 @@ API:
         RUNNER:
           - aws/fedora-38-x86_64
           - aws/fedora-39-x86_64
+      - IMAGE_TYPE: ["iot-bootable-container"]
+        RUNNER:
+          - aws/fedora-39-x86_64
```

Comment:
Ftr, this is targetting F40+. I know that supporting F39 is nice to have, but if this proves to be impossible, I'm happy with dropping it.

Reply:
The image builds fine on F39 in my own testing (and in osbuild/images) and bootc is available there so it should be fine.  If any issues come up that are only solvable on F40 I guess we can drop it.